metricq_wizard_backend/api/views/metric.py

```diff
@@ -104,6 +104,19 @@ def _get_interval_max_ms(interval_min_ms: int, interval_factor: int) -> int:
     return interval_min_ms * (interval_factor ** (n + 1))
 
 
+@routes.post("/api/metrics/forget")
+async def post_metrics_forget_metadata(request: Request):
+    client: Configurator = request.app["metricq_client"]
+    ids: list[metricq.Metric] = await request.json()
+
+    deleted = await client.forget_metrics(ids)
+
+    return json_response(
+        {"status": "ok" if len(deleted) == len(ids) else "error", "deleted": deleted},
+        status=200 if len(deleted) == len(ids) else 400,
+    )
```

Comment:
I don't like this on multiple accounts:
- the same condition checked twice
- multiple bulk if else make it hard to read
- "status": "error" is quite useless

I suggest to make a normal if/else here. Then you can go the extra mile and really test `set(deleted) == set(ids)` and write a better error message.